frontend/src/contexts/AuthContext.jsx

```diff
@@ -0,0 +1,22 @@
+import { createContext, useContext, useState, useMemo } from "react";
+import PropTypes from "prop-types";
+
+const AuthContext = createContext();
+
+export function AuthContextProvider({ children }) {
+  const [token, setToken] = useState();
+
+  const tokenManage = useMemo(() => {
+    return { token, setToken };
+  }, [token]);
+
+  return (
+    <AuthContext.Provider value={tokenManage}>{children}</AuthContext.Provider>
+  );
+}
+
+AuthContextProvider.propTypes = {
+  children: PropTypes.node.isRequired,
```

Comment:
`node` prop-type is used to render anything, but in your case that's a React element you want to render. So the correct type is `element` [Documentation](https://legacy.reactjs.org/docs/typechecking-with-proptypes.html#requiring-single-child)